.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,22 @@
+// Reference: https://code.visualstudio.com/docs/remote/containers#_devcontainerjson-reference
+{
+	"name": "www.graycore.io",
+	"dockerFile": "Dockerfile",
+	"onCreateCommand": [
+		"/bin/bash",
+		".devcontainer/onCreateCommand.sh"
+	],
+	"extensions": [
+		"stylelint.vscode-stylelint",
+		"naumovs.color-highlight",
+		"bierner.markdown-preview-github-styles",
+		"mrmlnc.vscode-scss",
+		"tehnix.vscode-tidymarkdown",
+		"dbaeumer.vscode-eslint"
+	],
```

Comment:
Why didn't you include `apollographql.vscode-apollo` in this list?

Reply:
That's a good addition. I'll add it.

Reply:
Done!